src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_StringConcat.cs

```diff
@@ -506,5 +816,24 @@ static bool isFieldOfMarshalByRef(BoundExpression expr, CSharpCompilation compil
                 return false;
             }
         }
+
+        private static MethodSymbol? FindSpecificToStringOfStructType(TypeSymbol exprType, MethodSymbol objectToStringMethod)
+        {
+            if (exprType.IsValueType && !exprType.IsTypeParameter())
+            {
+                var namedType = (NamedTypeSymbol)exprType;
```

Comment:
>namedType [](http://example.com/codeflow?start=20&length=9)

I am not going to block on this rename this time, but I am going to emphasize again, that any changes to the code for which we are not intending to change behavior should be avoided in PRs that do change behavior. Extracting a block of code into a reusable helper is not an intention to change behavior.